src/routes/notes.nim

```diff
@@ -12,6 +12,18 @@ proc createNotesRouter*(cfg: Config) =
     get "/i/notes/@id":
       let
         prefs = cookiePrefs()
+        path = getPath()
         article = await getGraphArticle(@"id")
-        note = renderNote(article, prefs)
+
+      let tweets = article
+        .entities
+        .filterIt(it.entityType == ArticleEntityType.tweet)
+        .mapIt(getTweet(it.tweetId))
+        .all
+        .await
+        .filterIt(it != nil)
+        .mapIt((it.tweet.id, it.tweet))
+        .toTable
```

Comment:
This Rust-style solution is very non-idiomatic and slow in Nim since `filterIt` and `mapIt` fully create the lists instead of it being an iterator chain.